src/main/java/edu/project2/SolutionMaze.java

```diff
@@ -0,0 +1,99 @@
+package edu.project2;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class SolutionMaze {
```

Comment:
```suggestion
public class MazeSolver {
```
более грамотное название с точки зрения английского)